frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -234,6 +234,25 @@ function combineLegacyRecordingFilters(
     }
 }
 
+function sortRecordings(recordings: SessionRecordingType[], order: RecordingsQuery['order']): SessionRecordingType[] {
+    const orderKey:
+        | 'recording_duration'
+        | 'active_seconds'
+        | 'inactive_seconds'
+        | 'console_error_count'
+        | 'click_count'
+        | 'keypress_count'
+        | 'mouse_activity_count'
+        | 'start_time' = order === 'duration' ? 'recording_duration' : order === 'random_sample' ? 'start_time' : order
```

Comment:
why is random sample mapped to start time here?

Reply:
The idea behind the randomisation was to pick a stratified set the were chosen over the course of your timespan. It felt weird looking at it that the results were unordered... which sounds funny to say 😅 

Maybe the right thing to do is have the random sample be a separate toggle and thus you can still "order" its results

Reply:
I'm not sure if this is more confusing (i.e. i asked for a random sample but these feel in order) or without. I like the separate toggle idea! Anyway will leave upto you to decide ⭐

Reply:
I implemented the toggle so let's see how that feels to folks!